ui.R

```diff
@@ -443,6 +443,79 @@ ui <- shinyUI(fluidPage(
                         )
                       )
              ),
+             tabPanel("Rentals",
+               fluidPage(
+                 div(class="title_container",
+                   h2("HEMM Rental Optimization")
+                 ),
+                 
+                 sidebarLayout(
+                   sidebarPanel(
+                     numericInput("rented_hemms", 
+                                 "Number of Rented HEMMs:",
+                                 min = 1, 
+                                 max = as.numeric(input$hemm_count),
```

Comment:
@sidraelx inside ui.R you can't use reactive values. Shift this inside the server.R and push one more commit, and tag me. See if you are not making similar mistakes in other places. And fix all. Did you run the code?
If you would have tried to run it you would have got this 
```
An error has occurred!
object 'input' not found
```